products/red-hat-openshift.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: OpenShift
+title: Red Hat OpenShift
 category: server-app
-iconSlug: openshift
-permalink: /openshift
+iconSlug: redhatopenshift
+permalink: /red-hat-openshift
```

Comment:
Add a few redirects to provide a few memorable URLs:

```suggestion
permalink: /red-hat-openshift
alternate_urls:
-   /openshift
-   /rh-openshift
```

Reply:
Good idea, but I think the iconSlug has to be the old value. See https://simpleicons.org/icons/redhatopenshift.svg